internal/llrp/reader_test.go

```diff
@@ -194,6 +195,7 @@ func dummyReply(h *Header, rfid net.Conn) error {
 // randomReply returns a function that sends replies with a random payload
 // such that minSz <= len(payload) <= maxSz.
 func randomReply(minSz, maxSz uint32) func(h *Header, rfid net.Conn) error {
+	//nolint: gosec
 	rnd := rand.New(rand.NewSource(1))
```

Comment:
What does linter not like here?

Reply:
`internal/llrp/reader_test.go:197:9: G404: Use of weak random number generator (math/rand instead of crypto/rand) (gosec)
        rnd := rand.New(rand.NewSource(1))
               ^
internal/llrp/reader_test.go:395:10: G404: Use of weak random number generator (math/rand instead of crypto/rand) (gosec)
                        sz := rand.Int31n(1024)
                              ^
internal/llrp/reader_test.go:397:4: G404: Use of weak random number generator (math/rand instead of crypto/rand) (gosec)
                        rand.Read(data)`

Reply:
ok, we include the `G404: Use of weak random number generator` description in the `// nolint: gosec` line so other know why. Like this:
```
//nolint:gosec // G404: Use of weak random number generator
```

Reply:
I have added the comments mentioned